From 07fa07278d0e71abea250f770ee6e1beffaef971 Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Wed, 3 Apr 2019 17:58:49 +0200
Subject: GCC: do not use COMPILER_GCC

GCC chromium build fails because of missing declaration of
COMPILER_GCC. So avoid that dependency and check directly
__GNUC__.

diff --git a/src/common/debug.h b/src/common/debug.h
index 64783687b..8c57f58be 100644
--- a/src/common/debug.h
+++ b/src/common/debug.h
@@ -248,7 +248,7 @@ std::ostream &FmtHex(std::ostream &os, T value)
 #    define EVENT(message, ...) (void(0))
 #endif
 
-#if defined(COMPILER_GCC) || defined(__clang__)
+#if defined(__GNUC__) || defined(__clang__)
 #    define ANGLE_CRASH() __builtin_trap()
 #else
 #    define ANGLE_CRASH() ((void)(*(volatile char *)0 = 0)), __assume(0)
-- 
2.19.1

